packages/stores/types.ts

```diff
@@ -86,6 +86,15 @@ type TTradingPlatformAvailableAccount = {
     sub_account_type: string;
 };
 
+type TAvailableCFDAccounts = {
+    availability: 'Non-EU' | 'EU' | 'All';
```

Comment:
There are some repeated types at the previous changes as well. Can we combine it here instead of declaring it everywhere?

Reply:
it is not used so removing it.